console/src/main/scripts/plugins/metrics/ts/app-details/appServerDatasourcesDetails.ts

```diff
@@ -240,6 +224,27 @@ module HawkularMetrics {
       this.getDrivers();
     }
 
+    private getAlerts(res:any):void {
+      let dsArray:IAlert[];
+      let promise = this.HawkularAlertsManager.queryAlerts({
+        statuses: 'OPEN',
+        tags: 'resourceId|' + res.id,
+        startTime: this.startTimeStamp,
+        endTime: this.endTimeStamp
+      }).then((data)=> {
+        _.forEach(data.alertList, (item) => {
+          item['alertType'] = item.context.alertType;
```

Comment:
It is preferable to type the item(IAlert) then use item.alertType instead of item['alertType'] which is not typed. This just helps to cuts down on typos and stupid little errors in the long run.